ServerCore/ModelBases/EventSpecificPageModel.cs

```diff
@@ -214,7 +214,7 @@ public async Task BindModelAsync(ModelBindingContext bindingContext)
         }
 
         public string GetFileStoragePrefix() {
-            return FileManager.GetFileStoragePrefix(Event.ID, "");
+            return FileManager.GetFileStoragePrefix((Event?.ID ?? 0), "");
```

Comment:
How did you get a null event on event specific pages? That shouldn't happen

Reply:
If you navigate to a page for an event that doesn't exist, like https://puzzlehunt.azurewebsites.net/pd2025/play due to a typo or something, the website crashes because the event object doesn't exist.  It's not a good look, so this at least lets it finish loading and shows our boilerplate which is actionable.  From a user standpoint, this is a much more friendly error state: 
![image](https://github.com/PuzzleServer/mainpuzzleserver/assets/30329717/019a3437-4194-4921-8eb9-aab86ed3f0b1)